src/app/views/panels/widgets/mapping_editor.tsx

```diff
@@ -399,7 +418,7 @@ export class MappingEditor extends React.Component<
             );
           } else {
             return (
-              <span className="el-mapping-scale">
+              <span>
```

Comment:
Did you intend on removing this className?

Reply:
No, I didn't intend to remove it. Good catch!